### ISSUE TITLE
Exercise 14.17

### ISSUE THREAD

**Issue by tonicanada** [pim-book/exercises#17]
I'm stuck on exercise 14-17, and I don't know how to proceed. This exercise involves transforming the computational graph to be able to detect digits from 0 to 9, not just 0 and 1. I converted the labels to a 'one hot' format, created the SoftMax node and CrossEntropyErrorNode, but I can't get it to work. Can you give me some hints to solve it. Here I attach the 2 files.

[acm_neural_network_softmax.zip](https://github.com/pim-book/exercises/files/12870153/acm_neural_network_softmax.zip)

Thank you in advance!
